src/app/components/client/Chart.tsx

```diff
@@ -59,91 +74,141 @@ export const DoughnutChart = (props: Props) => {
+          >
+            <circle
+              cx={diameter / 2}
+              cy={diameter / 2}
+              r={radius}
+              fill="none"
+              strokeWidth={ringWidth}
+              className={styles.gutter}
+            />
+            {slices}
+            {l10n.getFragment("exposure-chart-heading", {
+              elems: {
+                nr: (
+                  <text
+                    className={styles.headingNr}
+                    fontSize={headingNumberSize}
+                    x={diameter / 2}
+                    y={diameter / 2 - headingGap / 2}
+                    textAnchor="middle"
+                  />
+                ),
+                label: (
+                  <text
+                    className={styles.headingLabel}
+                    fontSize={headingLabelSize}
+                    x={diameter / 2}
+                    y={diameter / 2 + headingLabelSize + headingGap / 2}
+                    textAnchor="middle"
+                  />
+                ),
+              },
+              vars: { nr: sumOfFixedExposures },
+            })}
+          </svg>
+          <div className={styles.legend}>
+            <table>
+              <thead>
+                <tr>
+                  {/* The first column contains the chart colour,
+                      which is irrelevant to screen readers. */}
+                  <td aria-hidden={true} />
+                  <th>
+                    {l10n.getString("exposure-chart-legend-heading-type")}
+                  </th>
+                  <th>{l10n.getString("exposure-chart-legend-heading-nr")}</th>
                 </tr>
-              ))}
-            </tbody>
-          </table>
+              </thead>
+              <tbody>
+                {props.data.map(([label, num]) => (
+                  <tr key={label}>
+                    <td aria-hidden={true}>
+                      <svg viewBox="0 0 10 10">
+                        <rect rx={2} width="10" height="10" />
+                      </svg>
+                    </td>
+                    <td>{label}</td>
+                    <td>
+                      {l10n.getString("exposure-chart-legend-value-nr", {
+                        nr: num,
+                      })}
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+            </table>
+          </div>
         </div>
-      </div>
-      <figcaption>
-        This chart shows the total number of exposures that are fixed (
-        {sumOfFixedExposures} out of {props.totalExposures}).
-      </figcaption>
-    </figure>
+        <figcaption>
+          {l10n.getString("exposure-chart-caption")}
+          <button
+            aria-label={l10n.getString("modal-open-alt")}
+            onClick={() => explainerDialogState.open()}
+          >
+            <QuestionMarkCircle
+              width="15"
+              height="15"
+              alt={l10n.getString("modal-open-alt")}
+            />
```

Comment:
Not sure if it matters, but noticed sometimes the `<QuestionMarkCircle>` is 13&times;13, other times it's 15&times;15:

```sh
git grep -n -A3 "<QuestionMarkCircle" | grep -E "(QuestionMarkCircle|width=|height=)"

src/app/components/client/Chart.tsx:189:            <QuestionMarkCircle
src/app/components/client/Chart.tsx-190-              width="15"
src/app/components/client/Chart.tsx-191-              height="15"

src/app/components/client/ExposureCard.tsx:180:          icon={<QuestionMarkCircle alt="" width="13" height="13" />}

src/app/components/client/ExposureCard.tsx:223:            icon={<QuestionMarkCircle alt="" width="13" height="13" />}

src/app/components/client/ExposureCard.tsx:234:            icon={<QuestionMarkCircle alt="" width="13" height="13" />} // default icon for categories without a unique one

src/app/components/client/ExposuresFilter.tsx:238:              <QuestionMarkCircle
src/app/components/client/ExposuresFilter.tsx-239-                width="15"
src/app/components/client/ExposuresFilter.tsx-240-                height="15"

src/app/components/client/ExposuresFilter.tsx:254:              <QuestionMarkCircle
src/app/components/client/ExposuresFilter.tsx-255-                width="15"
src/app/components/client/ExposuresFilter.tsx-256-                height="15"

src/app/components/client/ProgressCard.tsx:119:          <QuestionMarkCircle alt="" width="15" height="15" />
```